public/kibi_timeline.js

```diff
@@ -149,25 +149,29 @@ define(function (require) {
         const groupId = group.id;
         const groupColor = group.color;
 
+        searchSource.highlight({
```

Comment:
Not sure if this provides any value. The use of kibana's highlight tags is in an angular filter. This PR needs to pull highlighted terms prior to display on the view.

Also, using the default `<em>` tag requires less data to be sent from server to client.